target/linux/ipq806x/image/Makefile

```diff
@@ -81,6 +81,21 @@ define Device/ZyXELImage
 	IMAGE/sysupgrade.bin/squashfs := append-rootfs | pad-to $$$$(BLOCKSIZE) | sysupgrade-tar rootfs=$$$$@ | append-metadata
 endef
 
+define Device/askey_rt4230w
```

Comment:
This should be
askey_rt4230w-rev6
then.

The compatible in DTS and the base-files case entries should be updated to askey,rt4230w-rev6 then.